tests/switchless/switchless.edl

```diff
@@ -3,18 +3,28 @@
 
 enclave {
     trusted {
-        public int enc_echo(
+        public int enc_echo_switchless(
```

Comment:
I thought there was an EDL keyword for switchless, why isn't that used?

Reply:
This is a regular ECALL. The `switchless` in the name simply means it will call into a switchless OCALL, in contrast to `enc_echo` which calls into a regular OCALL.